GB/TicTacToe Game/settings/game_settings.py

```diff
@@ -0,0 +1,33 @@
+import sys
+from core.model import TicTacToe
+from settings.constants import *
+

```

Comment:
Попробуйте добавить это
```suggestion
sys.path.append('..')
```